### PR TITLE
Viz - various improvements

### PR DESCRIPTION
**Pull request by kumavis**
- [browserify] re-add `policyDebug` option
- [viz] vr: epic perf fix
- [viz] vr: package label on hover

these changes were largely resurrected from some old incomplete viz work sessions

### CODE REVIEW

**Comment by kumavis** [comment]
@weizman unexpected scuttling related test failure here

**Comment by weizman** [comment]
> @weizman unexpected scuttling related test failure here

Just forgot to update policy.json #442 (merge to fix).

Also, there is a new file `xyz.js` in the root dir, in case you added it by mistake.

**Comment by weizman** [comment]
> @weizman unexpected scuttling related test failure here

@kumavis  Just forgot to update policy.json #442 (merge to fix).

Also, there is a new file `xyz.js` in the root dir, in case you added it by mistake.

**Comment by kumavis** [comment]
@weizman oh thanks!

the error message made me think it was scuttling related
```
 Error: LavaMoat - property "Math" of globalThis is inaccessible under scuttling mode. To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.
  at get (testBundle.js:11205:19)
  at Object.<anonymous> (testBundle.js:11723:35)
  at Object.<anonymous> (testBundle.js:11725:4)
  at internalRequire (testBundle.js:11265:27)
  at requireRelative (testBundle.js:11308:27)
  at requireRelativeWithContext (testBundle.js:11277:18)
  at Object._defineEntry (testBundle.js:11701:24)
  at Object.<anonymous> (testBundle.js:11703:6)
  at internalRequire (testBundle.js:11265:27)
  at runModule (testBundle.js:11679:12)
error Command failed with exit code 1.
```

and good eye on the extra file

**Comment by weizman** [comment]
> @weizman oh thanks!
> 
> the error message made me think it was scuttling related
> 
> ```
>  Error: LavaMoat - property "Math" of globalThis is inaccessible under scuttling mode. To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.
>   at get (testBundle.js:11205:19)
>   at Object.<anonymous> (testBundle.js:11723:35)
>   at Object.<anonymous> (testBundle.js:11725:4)
>   at internalRequire (testBundle.js:11265:27)
>   at requireRelative (testBundle.js:11308:27)
>   at requireRelativeWithContext (testBundle.js:11277:18)
>   at Object._defineEntry (testBundle.js:11701:24)
>   at Object.<anonymous> (testBundle.js:11703:6)
>   at internalRequire (testBundle.js:11265:27)
>   at runModule (testBundle.js:11679:12)
> error Command failed with exit code 1.
> ```

Yea I'm pretty sure that is an error message being thrown on purpose by the scuttling test, not sure why it is being printed out in the process..

File: packages/browserify/src/index.js [hunk 1]
```diff
@@ -111,19 +111,19 @@ function getConfigurationFromPluginOpts (pluginOpts) {
     pr: 'includePrelude',
     prelude: 'includePrelude',
     pp: 'prunePolicy',
-    prunepolicy: 'prunePolicy',
```
Comment (legobeat) [inline packages/browserify/src/index.js:114]: Are the browserify changes intimately linked to the viz changes? If not can they be broken out to a separate PR?

**Comment by legobeat** [comment]
https://github.com/LavaMoat/LavaMoat/actions/runs/3888067554/jobs/6635015908
```
 [2023-01-10 23:15:13] [build-stderr] java.lang.OutOfMemoryError: Java heap space
```
:shrug:

**Review by naugtur — APPROVED** [review]
